community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/slurmsync.py

```diff
@@ -83,10 +83,9 @@
 )
 
 
-def start_instance_op(inst, project=None):
-    project = project or lkp.project
-    return compute.instances().start(
-        project=project,
```

Comment:
Why remove the additional parameter?  Is it not possible to create VMs in a separate project from the controller?

(This question applies to the other locations where you removed project as a parameter)

Reply:
I removed all optional that never used. This is technical debt.